src/AltinnCore/Common/Services/Implementation/InstanceAppSI.cs

```diff
@@ -72,16 +72,16 @@ public async Task<Instance> InstantiateInstance(StartServiceModel startServiceMo
             Guid instanceId;
             Instance instance = null;
             string org = startServiceModel.Org;
-            string appId = ApplicationHelper.GetFormattedApplicationId(org, startServiceModel.Service);
-            string appName = startServiceModel.Service;
+            string app = startServiceModel.Service;
+            string appId = ApplicationHelper.GetFormattedApplicationId(org, app);
```

Comment:
I would suggest renaming this parameter as app is also the id of the application in some ways. Maybe formattedAppId? or somehing along those lines?